python/ray/experimental/channel/torch_tensor_type.py

```diff
@@ -58,6 +59,10 @@ def __init__(
                 shape; if it does not match, the task will error.
             dtype: The expected dtype of the torch.Tensor. Similar to the
                 shape, this may be statically or dynamically declared.
+            direct_return: Whether the tensor is sent directly or inside of
+                other metadata. For GPU-GPU channels, this allows the sender
```

Comment:
This feels a little leaky abstraction to me. what's the benefit of removing additional channel? Maybe we should just mention the benefits? (is it improving the perf? Reducing memory overhead?)

Reply:
Yes, it is for performance, will update the comment.